### PR TITLE
fix main bug

### PR DESCRIPTION
**Pull request by Goven112**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/userPage/ActiveMembership/PlastDegree/FormAddPlastDegree.tsx [hunk 1]
```diff
@@ -44,13 +54,22 @@ const FormAddPlastDegree = ({
       plastDegreeId: info.plastDegree,
       dateStart: info.datepickerStart._d,
       userId: userId,
+      
     };
     setVisibleModal(false);
     visiableDegree.current = false;
+    visiableCities.current = false;
+  
+    const cityMember1 : CityMember =  (await addFollowerWithId(info.city,userId)).data;
```
Comment (ostappppppp) [inline src/pages/userPage/ActiveMembership/PlastDegree/FormAddPlastDegree.tsx:63]: do you think `cityMember1` is appropriate name for variable?

File: src/pages/userPage/ActiveMembership/PlastDegree/FormAddPlastDegree.tsx [hunk 2]
```diff
@@ -44,13 +54,22 @@ const FormAddPlastDegree = ({
...
+    await toggleMemberStatus(cityMember1.id);
+
     await activeMembershipApi.postUserPlastDegree(userPlastDegreePost);
     handleAddDegree();
     form.resetFields();
     resetAvailablePlastDegree();
+   
+    if( UpdateData){  await UpdateData()};
```
Comment (ostappppppp) [inline src/pages/userPage/ActiveMembership/PlastDegree/FormAddPlastDegree.tsx:71]: format code
if you want use condition statements in one line, consider using `ternary` operator

File: src/pages/userPage/ActiveMembership/PlastDegree/FormAddPlastDegree.tsx [hunk 3]
```diff
@@ -61,19 +80,24 @@ const FormAddPlastDegree = ({
     notificationLogic("success", successfulAddDegree());
   };
 
-  const handleOnChange = (value: any) => {
+  const handleOnChange = async (value: any) => {
     if (value === "Пластприят") {
```
Comment (ostappppppp) [inline src/pages/userPage/ActiveMembership/PlastDegree/FormAddPlastDegree.tsx:84]: I see that you using that word `Пластприят` at least five times
consider to move it in some enum(I talk about all harcoded strings)

**Review by Omeliukh-Nazar — APPROVED** [review]

**Review by NatalyMinziuk — APPROVED** [review]

**Review by bkondrat — APPROVED** [review]

File: src/pages/userPage/ActiveMembership/PlastDegree/FormAddPlastDegree.tsx [hunk 4]
```diff
@@ -47,10 +56,18 @@ const FormAddPlastDegree = ({
     };
     setVisibleModal(false);
     visiableDegree.current = false;
+    visiableCities.current = false;
```
Comment (ostappppppp) [inline src/pages/userPage/ActiveMembership/PlastDegree/FormAddPlastDegree.tsx:59]: should be `visibleCities`

File: src/pages/userPage/ActiveMembership/PlastDegree/FormAddPlastDegree.tsx [hunk 5]
```diff
@@ -47,10 +56,18 @@ const FormAddPlastDegree = ({
...
+  
+    const newCityFollower : CityMember =  (await addFollowerWithId(info.city,userId)).data;
+    await toggleMemberStatus(newCityFollower.id);
     await activeMembershipApi.postUserPlastDegree(userPlastDegreePost);
     handleAddDegree();
     form.resetFields();
     resetAvailablePlastDegree();
+    if( UpdateData)
```
Comment (ostappppppp) [inline src/pages/userPage/ActiveMembership/PlastDegree/FormAddPlastDegree.tsx:67]:
```suggestion
    if(UpdateData)
```

**Pull request merged**